resources/views/emails/internal/memberOnEventNotification.blade.php

```diff
@@ -3,12 +3,10 @@
 </p>
 
 <p>
-	Zojuist heeft <a href="{{ url('/members', $member->id) }}">{{ $member->voornaam }} {{ $member->tussenvoegsel }} {{ $member->achternaam }}</a> zich via {{ ($member->geslacht=='M') ? 'zijn' : 'haar' }} profiel aangemeld voor <a href="{{ url('/events', $event->id) }}">{{ $event->naam }}</a> ({{ $event->code }})!
+	Zojuist heeft <a href="{{ url('/members', $member->id) }}" target="_blank">{{ $member->volnaam }}</a> zich via {{ ($member->geslacht=='M') ? 'zijn' : 'haar' }} profiel aangemeld voor <a href="{{ url('/events', $event->id) }}" target="_blank">{{ $event->naam }}</a> ({{ $event->code }})!
```

Comment:
try to avoid using 'haar' / 'zijn' . Currently if the gender is unknown. it will default to 'haar' . You can rewrite the sentence to not include this.